src/harness.ts

```diff
@@ -284,6 +285,19 @@ export class Harness<P extends WidgetProperties, W extends Constructor<WidgetBas
 	}
 
 	/**
+	 * Call a listener on a target node of the virtual DOM.
+	 * @param method The method to call on the target node
+	 * @param options A map of options that effect the behaviour of `callListener`
```

Comment:
behavior (I think we're trying to be consistent in US English in Dojo 2 docs, though I don't personally care)